### DISCUSSION TITLE
zh/guide/drivers/develop

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#51] [Announcements]
# zh/guide/drivers/develop

AList v3的文档

https://alist.nn.ci/zh/guide/drivers/develop.html

**Comment by CNBpanda** [comment]
大佬，可以帮忙加上小飞机网盘的存储吗，不知道好不好弄，官网是https://www.feijipan.com/，如果有空方便帮忙看下，感谢大佬。

**Reply by tianwenyuyu** [reply]
同求

**Comment by kingfly11** [comment]
有没有大佬能弄个FLOWUS 息流的网盘,不限速,会员无限空间,就是小弟对编程一窍不通,希望有大佬能弄出来,谢谢

**Comment by Marsdancer** [comment]
坚果云网盘可以考虑下，官网 https://www.jianguoyun.com/

**Reply by 27810617** [reply]
webadv就行

**Reply by xuanwuwl** [reply]
坚果云官方支持webadv，可以通过webadv的形式添加

**Comment by SeptemberZero** [comment]
能添加棱束链的个人存储吗
官网https://www.lingshulian.com/

**Reply by gradendripar** [reply]
这个兼容S3协议 使用对象存储方式添加就行了

**Comment by cocoot** [comment]
https://pan.bitqiu.com/
比特球网盘支持一下吗?

**Reply by linzxc2010** [reply]
大佬这个能弄吗

**Reply by linzxc2010** [reply]
大佬关注下这个网盘

**Comment by liu822629502** [comment]
大佬可以帮忙看看微云网盘么，这qq号码的网盘可以搞上去么

**Comment by Steven7559** [comment]
能不能加一下腾讯微云 https://www.weiyun.com/

**Comment by vivi181** [comment]
可以加入dropbox吗？这个网盘还蛮常见的

**Comment by duansix** [comment]
最近很火的小龙云盘 大佬有空看看
https://www.stariverpan.com/download.html

**Reply by fususususu** [reply]
有谁弄了吗？

**Reply by 27810617** [reply]
跑路了

**Comment by 2003wxc** [comment]
加小飞机网盘、flow us、棱束链这三个吧

**Comment by abullstar** [comment]
大佬，能带上UC网盘吗？https://drive.uc.cn/，不胜感激。

**Comment by githuburanus** [comment]
请问可以通过捐赠方式有偿expand网盘吗？网盘是魔盘网络http://www.moopan.top/，感谢！

**Comment by 23343796498** [comment]
有大佬加一个cloudreve的存储可以吗，不知道好不好弄，麻烦大佬了

**Reply by clearng** [reply]
cloudreve的话可以用webdav啊

**Comment by BingoKingo** [comment]
大佬，求个微云挂载+1

**Comment by mtk2025** [comment]
有偿求加https://tmp.link/ 钛盘，q285742765

**Comment by fususususu** [comment]


**Reply by mtk2025** [reply]
你会加吗

**Comment by 23343796498** [comment]
你是？


Esacpe°
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "alist-org/docs"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年2月26日(星期天) 晚上7:11
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [alist-org/docs] zh/guide/drivers/develop (Discussion #51)





 不会，都哪来我邮箱的？奇了怪了。我也没有说过我会呀？ 
 
 
 
 
 
 
 3309843302 
 ***@***.*** 
 
 
 
 &amp;nbsp; 
 
 
 
 
 ------------------&amp;nbsp;原始邮件&amp;nbsp;------------------ 
 发件人:                                                                                                                        "alist-org/docs"                                                                                    ***@***.***&amp;gt;; 
 发送时间:&amp;nbsp;2023年2月26日(星期天) 晚上7:09 
 ***@***.***&amp;gt;; 
 ***@***.******@***.***&amp;gt;; 
 主题:&amp;nbsp;Re: [alist-org/docs] zh/guide/drivers/develop (Discussion #51) 
 
 
 
 
 
   
 你会加吗

**Comment by 23343796498** [comment]
你他妈谁啊？


Esacpe°
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "alist-org/docs"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年2月26日(星期天) 晚上7:09
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [alist-org/docs] zh/guide/drivers/develop (Discussion #51)





 
你会加吗

**Comment by iog666** [comment]
可以帮忙添加飞鸟嘛

**Comment by CNBpanda** [comment]
Me Too
发件人: fususususu
发送时间: 2023年2月26日 19:11
收件人: alist-org/docs
抄送: Dreamo; Comment
主题: Re: [alist-org/docs] zh/guide/drivers/develop (Discussion #51)

不会，都哪来我邮箱的？奇了怪了。我也没有说过我会呀？ 






3309843302 
***@***.*** 



&nbsp; 




------------------&nbsp;原始邮件&nbsp;------------------ 
发件人: "alist-org/docs" ***@***.***&gt;; 
发送时间:&nbsp;2023年2月26日(星期天) 晚上7:09 
***@***.***&gt;; 
***@***.******@***.***&gt;; 
主题:&nbsp;Re: [alist-org/docs] zh/guide/drivers/develop (Discussion #51) 






你会加吗

**Comment by 6XiaoXin6** [comment]
联通云盘还蛮多人用的 可支持下 .

**Reply by sunlyc** [reply]
是的，联通云盘有个联通云硬盘，可以挂载本地，可以获取到一些密钥吧

**Comment by zhushen12580** [comment]
`腾讯企业微信`  微盘可以试下哦

**Reply by zhushen12580** [reply]
开发文档：https://developer.work.weixin.qq.com/document/path/96844

**Comment by wujiaren** [comment]
Link方法返回的url对应的下载逻辑有什么规范吗，我按照模板加上自己公司的网盘，用raidrive挂载到本地，打开10M以上的文本下载会报错：远程主机强迫关闭了一个现有的连接，并且打开后是乱码的。

**Reply by wujiaren** [reply]
是否需要支持分段下载

**Comment by sunlyc** [comment]
希望加上联通云盘！

**Comment by csbxd** [comment]
能支持一下git协议吗？

**Comment by 863683** [comment]
playbook网盘，www.playbook.com 目前只有网页版，没有客户端。
注册提供100G基础空间，通过艺术家认证(门槛非常低)，扩容到4T。
请看这个网盘有搞头么，谢谢

**Comment by ckslvzyn** [comment]
曲奇云盘，2t容量，网址https://www.quqi.com/，安利一下呗

**Comment by ckslvzyn** [comment]
曲奇云盘，2t容量，网址https://www.quqi.com/，alist安利一下呗






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "alist-org/docs"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年8月16日(星期三) 下午4:22
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [alist-org/docs] zh/guide/drivers/develop (Discussion #51)





 邮件收悉，祝好！----------------------------------汪浩&nbsp;Wang Hao浙江大学管理学院浙江省杭州市西湖区余杭塘路866号E-mail: ***@***.*** ; ***@***.***: (+86)17602308292

**Comment by LailuoA77** [comment]
有没有方法加入挂载镜像源的方法

**Comment by azhango** [comment]
考虑一下添加一个S3/S4存储[https://www.dogecast.com/](https://www.dogecast.com/)

**Reply by TraderWagon** [reply]
本来就有

**Comment by dumblamb1** [comment]
大佬，WPS云文档，能否加入，这个有挑战性，多谢！

**Comment by yaoyunhan181** [comment]
大佬能帮忙加上城通网盘分享的存储吗？官网https://ctfile.com/，谢谢大佬。

**Comment by iszmxw** [comment]
能加上 github 仓库存储方式么

**Comment by tianwenyuyu** [comment]
求大佬弄个小飞机的储存方式 https://www.feijipan.com

**Comment by wucd18** [comment]
大佬，telebox可以添加吗？可以rclone，但是想挂载到alist。官网: https://www.linkbox.to/api-docs，感谢大佬！

**Comment by narouss** [comment]
求一个wps网盘驱动

**Comment by SouperGeng** [comment]
大佬啊  阿里相册可以帮忙加一下吗

**Comment by niaoquan** [comment]
大佬可以看看钛盘https://www.tmp.link/好做吗，虽然是临时网盘但是有访问就可以续时间有时会用到

**Comment by amlkiller** [comment]
有大佬添加云中转的存储吗？网址是： yzzpan.com

**Comment by xnz233** [comment]
永硕E盘支持一下吗?QAQ

**Comment by CaptainKiana** [comment]
请问大佬能支持uc网盘吗？

**Reply by CaptainKiana** [reply]
不好意思，之前没有看到

**Comment by xrcq** [comment]
大佬，可以帮忙加上城通网盘的存储吗，不知道好不好弄，官网是https://home.ctfile.com

**Comment by RzdPai** [comment]
大佬可不可以添加MediaFire

**Comment by mhqschen** [comment]
期待有大佬能搞定myqnapcloud storage

**Comment by briand2** [comment]
大佬，小飞机网盘目前只能挂载，但是不能下载，可以帮忙处理一下吗？

**Comment by 458722** [comment]
哪个大佬 能加一下 兔子 这个 超大型官方 https://bunny.net/

**Comment by zouyelin** [comment]
麻烦添加一下UC网盘https://drive.uc.cn

**Comment by 17812279363** [comment]
有大佬 搞一个飞书的  超大文件上传和下载吗

**Comment by TraderWagon** [comment]
https://www.febbox.com/

**Comment by xianger2017** [comment]
弄一个QQ邮箱中转站的~

**Comment by kingshirtAA** [comment]
添加   联想乐云 不限速很不错的

**Comment by jywcheng** [comment]
请问有人能做一下huang1111网盘的吗？

**Comment by abxup** [comment]
希望添加 欢太云盘，OPPO手机云盘

**Comment by 4873** [comment]
大佬能不能加个百度网盘群文件，考研资料很多在群里面

**Comment by 18810579687** [comment]
大佬可不可以添加以下这个地址,不知道是什么网盘但好像是安卓驱动：https://file.cnxiaobai.com/

**Comment by jqap** [comment]
哪位大佬搞一下蓝奏云分享链接的挂载呀？求求了！

**Comment by 1191533749** [comment]
### 亿安网盘
#### 请添加一下亿安网盘，网址是 https://dav.yiandrive.com/login 这款网盘看起来还挺不错。

**Comment by 1191533749** [comment]
### 亿安网盘
#### 请添加一下亿安网盘，网址是 https://dav.yiandrive.com/login 这款网盘看起来还挺不错。

**Reply by 1191533749** [reply]
#### 这个是亿安网盘官网 https://pan.yiandrive.com/   上面那个地址是网盘入口
